buildpacks/nodejs-engine/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Added node version 12.22.12.
```

Comment:
(nit) Does the automation need to add an extra newline here?

Reply:
I chose not to in case there was already something in the `[Unreleased]` section. Something like this would look weird:

```md
## [Unreleased]

- Added foo

- Changed a thing

## [1.0.0]

- blah
```

Reply:
Ah yes